testing-utils/src/lib/setup/console.mock.ts

```diff
@@ -7,15 +7,15 @@ let consoleErrorSpy: SpyInstance | undefined;
 beforeEach(() => {
   // In multi-progress-bars, console methods are overriden
```

Comment:
Right, this 😅 let me explain how I understood it.

`multi-progress-bars` overrides the whole `console` object so it no longer has `info` or `warn` methods. That is why I had to assert the existence of the methods before passing the spy and mock implementation.

Since latest version came out a year ago, unfortunately this behaviour has not changed. We can perhaps get rid of the conditions if we choose a different 3rd-party library that is not so invasive.